src/util/plain_to_request_mapper.ts

```diff
@@ -323,6 +323,8 @@ export class PlainToRequestMapper {
         map.set(KolibriRequestMethods.SubscribeRequestMethod, cV32.toSubscribeRequest);
         map.set(KolibriRequestMethods.NodeDeleteHistoryRequestMethod, cV32.toNodeDeleteHistoryRequest);
         map.set(KolibriRequestMethods.PermissionUserListRequestMethod, cV32.toPermissionUserListRequest);
+        map.set(KolibriRequestMethods.NodeSubscribeRequestMethod, cV33.toNodeSubscribeRequest);
```

Comment:
This feature is only available in the V3.3 protocol version. Mappings should be removed from the v3.2. mapper.

Reply:
Done